test/queries/simpleSelectWhere.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     find  : {
 
       // The queryString we expect to be rendered after calling `Sequel.select()`
-      queryString    : 'SELECT `foo`.`color`, `foo`.`id`, `foo`.`createdAt`, `foo`.`updatedAt`, `foo`.`bar`, `foo`.`bat`, `foo`.`baz` FROM `foo` AS `foo`  WHERE LOWER(`foo`.`color`) = "blue" ',
+      queryString    : "SELECT `foo`.`color`, `foo`.`id`, `foo`.`createdAt`, `foo`.`updatedAt`, `foo`.`bar`, `foo`.`bat`, `foo`.`baz` FROM `foo` AS `foo`  WHERE LOWER(`foo`.`color`) = 'bl\\'ue' ",
```

Comment:
This isn't valid SQL, I don't think.
`'bl\\'ue'` appears to mysql as the closed string literal `bl\` followed by some more SQL, `ue'`.  The first slash escapes the second slash, and nothing escape the proceeding single-quote.